packages/schematics/src/utils/module-utils.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
```

Comment:
Why is this separate from ast-utils? Most of the helpers there are module-related as well.

Reply:
Merged.